tests/test_utils.py

```diff
@@ -1,8 +1,65 @@
 import datetime
+import json
+import mock
 
-from django.test import SimpleTestCase
+from django.test import SimpleTestCase, override_settings
 
 from unicef_vision import utils
+from unicef_vision.loaders import VISION_NO_DATA_MESSAGE
+
+FAUX_VISION_URL = 'https://api.example.com/foo.svc/'
+FAUX_VISION_USER = 'jane_user'
+FAUX_VISION_PASSWORD = 'password123'
+
+
+class TestGetDataFromInsight(SimpleTestCase):
+    @override_settings(VISION_URL=FAUX_VISION_URL)
```

Comment:
can we move this decorators on the class?

Reply:
I think yes, i have moved them